sites/kit.svelte.dev/svelte.config.js

```diff
@@ -1,11 +1,15 @@
-import adapter from '@sveltejs/adapter-vercel';
+import adapter from '@sveltejs/adapter-static';
```

Comment:
I guess this should be reverted before merging, but besides that (and the PR needing a rebase) this looks good to me